mod/cp_notifications/languages/fr.php

```diff
@@ -164,6 +164,7 @@
 	'cp_newsletter:heading:notify:cp_mention:plural' => "Personnes vous ont mentionné!", 
 
 
```

Comment:
Plateforme is female so 'Nouvelle' and without the s if i'm right with the other one